recipe/meta.yaml

```diff
@@ -12,16 +12,17 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
-    - toolchain
     - cmake
-    - zeromq 4.2.3
+    - {{ compiler('c') }}
```

Comment:
Indeed that does make more sense, although from reading the CMakeLists.txt and looking at the build logs it seems that no compiler is really necessary since all this does is install headers. A C++ compiler would be required to build the demos and tests, but this doesn't do that. Maybe this should drop the compiler requirement altogether?